From 0b18f9753f577b042806efcef7ca07ce9fb5c758 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 30 Jul 2018 10:00:05 +0300
Subject: [PATCH] board: habana: update arch timer clock

Our clock is 200MHz and not the CPU clock 1200MHz
---

diff --git a/board/habanalabs/dali/uboot_asic.fragment b/board/habanalabs/dali/uboot_asic.fragment
index 581f779..c5c804f 100644
--- a/board/habanalabs/dali/uboot_asic.fragment
+++ b/board/habanalabs/dali/uboot_asic.fragment
@@ -1,4 +1,4 @@
-CONFIG_SYS_CLK_FREQ=1200000000
+CONFIG_SYS_CLK_FREQ=200000000
 CONFIG_CONARGS="serial0:115200n8"
 CONFIG_BAUDRATE=115200
 CONFIG_BOOTDELAY=0
